### ISSUE TITLE
strange issue

### ISSUE THREAD

**Issue by x-vibe** [uchiafarmer/chia-tools#1]
Hi there, we conversed on chiaforum regarding this little tool and I only had time tonight to look at it in more detail.
I seem to be getting the below error. Also note that I added the DIR to chia plots add -d ... and restarted the harvester (this machine has no keys).

Is there anything I am missing? Also, I'd probably change the names of target and destination (sounds like the same thing) to "analysis folder" and "destination folder".. something like that anyway :-)

```
(venv) superplotter@superplotter:~/chia-tools$ plotfilter -n -t /mnt/sdv/CHIAPOOL/ -d /mnt/sdv/CHIATESTING/ -v
2021-07-28 18:44:17.128  =========================
2021-07-28 18:44:17.128  INFO: Dry run in progress
2021-07-28 18:44:17.129  =========================
2021-07-28 18:44:17.130  Target directory found: /mnt/sdv/CHIAPOOL/
2021-07-28 18:44:17.131  Destination directory found: /mnt/sdv/CHIATESTING/
2021-07-28 18:44:17.132  'chia-blockchain' virtual environment detected: proceeding
2021-07-28 18:44:17.404  WARNING: Target directory does not exist in 'chia plots' paths
2021-07-28 18:44:17.404  Adding target directory to 'chia plots'
2021-07-28 18:44:17.966  Error. could not verify target directory was added  to 'chia plots' paths
```

should I maybe not run this in the venv? I just assumed that would need to be open.
My setup is:

Ubuntu 20.04 LTS Server, 4 GB RAM, 2 cores, 4 threads, NVME

Thanks for the hard work on this!

**Comment by uchiafarmer** [comment]
Huh.  Thats weird. 

This error means that it tried to add '/mnt/sdv/CHIAPOOL/' to chia with `chia plots add -d "/mnt/sdv/CHIAPOOL/"` but when it ran `chia plots show` to confirm it, it couldn't find it on the list.

I'll have a look at the code.

**Comment by uchiafarmer** [comment]
ok I think I found the issue, it looks like when it is added to `chia` with `chia plots add -d /mnt/sdv/CHIAPOOL/` chia strips off the trailing `/`, so when my program tries to verify it is added it doesn't show because its missing the last `/`. 

Try running it as `plotfilter -n -t /mnt/sdv/CHIAPOOL -d /mnt/sdv/CHIATESTING -v`

**Comment by uchiafarmer** [comment]
FIXED: with [21b6808](https://github.com/uchiafarmer/chia-tools/commit/21b6808efa9d98169686e048a313d6927bd78623#diff-24abbb2d31f3c270fd25d666226815c98b13ad3b6baaecd9e51951da7b52721e)
